src/components/SocialBar/SocialBar.st.css

```diff
@@ -0,0 +1,64 @@
+:import {
+  -st-from: "../../common/formatters.st";
+  -st-named: color;
+}
+
+:import {
+  -st-from: "../IconButton/IconButton.st.css";
+  -st-default: IconButton;
+}
+
+/*Defaults*/
+:vars {
+  /*Defaults params goes here*/
+}
+
+/*Overrides*/
+:vars {
+  /*Deceleration of overridable placeholders*/
+}
+.root {
+  -st-states: theme(enum(light, dark));
+
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  height: 50px;
+}
+
+/* imported */
+.themed-icon {
+
+}
+
+/* imported */
+.themed-text {
+
+}
+
+.root:theme(light) .themed-text {
+  color: black;
+}
+
+.root:theme(dark) .themed-text {
+  color: white;
+}
+
+.root:theme(light) .themed-icon path,
+.root:theme(light) .icon .IconButton path {
+  fill: black !important;
+  stroke: none !important;
```

Comment:
Is the `!important` really necessary here?

Reply:
we have 
```css
.root:skin(line) .icon path {
    stroke: color(fallback(value(IconColor), value(DefaultColor)));
    fill: none;
}

.root:skin(full) .icon path {
    fill: color(fallback(value(IconColor), value(DefaultColor)));
    stroke: color(fallback(value(IconColor), value(DefaultColor)))  ;
}
```
in IconButton. and this selectors are more specific.